client/src/app/core/ui-services/base-filter-list.service.ts

```diff
@@ -36,6 +36,7 @@ export type OsFilterOptions = (OsFilterOption | string)[];
 export interface OsFilterOption {
     label: string;
     condition: OsFilterOptionCondition;
+    filteredItemsLength: number;
```

Comment:
should be either optional or default 0, since you manually set it to 0 all the time